clouddriver-docker/src/test/groovy/com/netflix/spinnaker/clouddriver/docker/registry/api/v2/client/DockerRegistryClientSpec.groovy

```diff
@@ -52,6 +52,58 @@ class DockerRegistryClientSpec extends Specification {
     TypedInput catalogTypedInput = new TypedByteArray("application/json", json.getBytes())
     Response catalogResponse = new Response("/v2/_catalog/",200, "nothing", Collections.EMPTY_LIST, catalogTypedInput)
     getCatalog(_,_,_) >> catalogResponse
+
+    String schemaJson = """{
+         "schemaVersion": 2,
+         "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+         "config": {
+            "mediaType": "application/vnd.docker.container.image.v1+json",
+            "size": 4405,
+            "digest": "sha256:fa8d22f4899110fdecf7ae344a8129fb6175ed5294ffe9ca3fb09dfca5252c93"
+         },
+         "layers": [
+            {
+               "mediaType": "application/vnd.docker.image.rootfs.diff.tar.gzip",
+               "size": 3310095,
+               "digest": "sha256:1ace22715a341b6ad81b784da18f2efbcea18ff7b4b4edf4f467f193b7de3750"
+            }
+         ]
+      }"""
+    TypedInput schemaV2Input = new TypedByteArray("application/json", schemaJson.getBytes())
+    Response schemaV2Response = new Response("/v2/{name}/manifests/{reference}",200, "nothing", Collections.EMPTY_LIST, schemaV2Input)
+    getSchemaV2Manifest(_,_,_,_) >> schemaV2Response
+
+    String configDigestContentJson = """{
+        "architecture": "amd64",
+        "config": {
+          "Hostname": "",
+          "Env": [
+            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+          ],
+          "Cmd": [
+            "/opt/app/server"
+          ],
+          "Image": "sha256:3862e8f6f860c732be3fe0c0545330f9573a09cf906a78b06a329e09f9dc7191",
+          "Volumes": null,
+          "WorkingDir": "",
+          "Entrypoint": null,
+          "OnBuild": null,
+          "Labels": {
+            "branch": "main",
+            "buildNumber": "1",
+            "commitId": "b48e2cf960de545597411c99ec969e47a7635ba3",
+            "jobName": "test"
+          }
+        },
+        "container": "fc1607ce29cfa58cc6cad846b911ec0c4de76d426de2b528a126e715615286bc",
+        "created": "2021-02-16T19:18:50.176616541Z",
+        "docker_version": "19.03.6-ce",
+        "os": "linux",
+        "rootfs": {}
+      }"""
```

Comment:
```suggestion
    String configDigestContentJson = '''{
        "architecture": "amd64",
        "config": {
          "Hostname": "",
          "Env": [
            "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
          ],
          "Cmd": [
            "/opt/app/server"
          ],
          "Image": "sha256:3862e8f6f860c732be3fe0c0545330f9573a09cf906a78b06a329e09f9dc7191",
          "Volumes": null,
          "WorkingDir": "",
          "Entrypoint": null,
          "OnBuild": null,
          "Labels": {
            "branch": "main",
            "buildNumber": "1",
            "commitId": "b48e2cf960de545597411c99ec969e47a7635ba3",
            "jobName": "test"
          }
        },
        "container": "fc1607ce29cfa58cc6cad846b911ec0c4de76d426de2b528a126e715615286bc",
        "created": "2021-02-16T19:18:50.176616541Z",
        "docker_version": "19.03.6-ce",
        "os": "linux",
        "rootfs": {}
      }'''
```

Reply:
hmmm, I can switch it to single quotes but units passed locally, and it doesn't seem to be the source of the failures. looks like Kubernetes integrations are the reason with the following error messages:

```
2021-02-25 21:45:40.154  INFO 2435 --- [    Test worker] .b.t.c.SpringBootTestContextBootstrapper : Neither @ContextConfiguration nor @ContextHierarchy found for test class [com.netflix.spinnaker.clouddriver.kubernetes.it.InfrastructureIT], using SpringBootContextLoader
2021-02-25 21:45:40.161  INFO 2435 --- [    Test worker] o.s.t.c.support.AbstractContextLoader    : Could not detect default resource locations for test class [com.netflix.spinnaker.clouddriver.kubernetes.it.InfrastructureIT]: no resource found for suffixes {-context.xml, Context.groovy}.
```